src/components/ae-fraction/ae-fraction.vue

```diff
@@ -0,0 +1,81 @@
+<template>
+  <span class="ae-fraction" :class="[ fill ]">
+    <sup>{{ numerator }}</sup>
+    &frasl;
+    <sub>{{ denominator }}</sub>
+  </span>
+</template>
+<script>
+export default {
+  name: 'ae-fraction',
+  props: {
+    /**
+     * Fill property changes the color state of the fraction component.
+     * Select something between: `primary, secondary, neutral, alternative, white, black`
+     */
+    fill: {
+      type: String,
+      validator: value => [
+        'primary',
+        'secondary',
+        'neutral',
+        'alternative',
+        'white',
+        'black',
```

Comment:
Isn't better to use more general colors for easier adjustments in future? For example, Bootstrap uses `light` and `dark`: https://getbootstrap.com/docs/4.2/components/alerts/

Reply:
These are fine, we're using this ones overall throughout the components.